docs/src/main/asciidoc/deploying-to-google-cloud.adoc

```diff
@@ -0,0 +1,236 @@
+* https://cloud.google.com/sdk[Cloud SDK CLI Installed]
+
+This guide will take as input an application developed in the link:getting-started[Getting Started guide].
+
+Make sure you have the getting-started application at hand, or clone the Git repository: `git clone {quickstarts-clone-url}`,
+or download an {quickstarts-archive-url}[archive]. The solution is located in the `getting-started` directory.
+
+== Login to Google Cloud
+
+Login to Google Cloud is necessary for deploying the application and it can be done as follows:
+
+[source, subs=attributes+]
+----
+gcloud auth login
+----
+
+== Deploying to Google Cloud Functions
+
+Quarkus support deploying your application to Google Cloud Functions via the following extensions:
+
+- link:gcp-functions[Google Cloud Functions]: Build functions using the Google Cloud Functions API.
+- link:gcp-functions-http[Google Cloud Functions HTTP binding]: Build functions using Quarkus HTTP APIs: RESTEasy (JAX-RS),
+Undertow (Servlet), Vert.x Web, or link:funqy-http[Funqy HTTP].
+- link:funqy-gcp-functions[Funky Google Cloud Functions]: Build functions using Funqy.
+
+Each extension support a specific kind of application development,
+follow the specific guides for more information on how to develop, package and deploy you application with them.
+
+== Deploying to Google App Engine Standard
+
+We will only covers Java 11 runtime as the Java 8 runtime uses its own Servlet engine which is not compatible with Quarkus.
+
+Before all, make sure to have an App Engine environment initialized for your Google Cloud project, if not, initialize one via `gcloud app create --project=[YOUR_PROJECT_ID]`.
+
+Then, you will need to create a `src/main/appengine/app.yaml` file, let's keep it minimalistic with only the selected engine:
+
+[source, yaml]
+----
+runtime: java11
+----
+
+This will create a default service for your App Engine application.
+
+Then, you can choose to build the application by yourself or letting `gcloud` or the Google Cloud Maven building it for you.
+
+=== Building the application manually
+
+Setup your application to be packaged as an uber-jar via your `application.properties` file:
+
+[source, properties]
+----
+quarkus.package.type=uber-jar
+----
+
+Use Maven to build the application using `mvn clean package`, it will generate a single JAR that contains all the classes of your application including its dependencies.
+
+Finally, use `gcloud` to deploy your application as an App Engine service.
+
+[source, shell script]
+----
+gcloud app deploy target/getting-started-1.0-SNAPSHOT-runner.jar
+----
+
+This command will upload your application jar and launch it on App Engine.
+
+When done, the output will display the URL of your application (target url), you can use it with curl or directly open it on your browser using `gcloud app browse`.
+
+As the getting started endpoint is located on the `/hello` path, make sure to navigate to it.
```

Comment:
Not sure this sentence helps much, maybe It should be removed

Reply:
By default when you type `app browse` or go to the URL written on the console you have the 404 page.

Reply:
Are you sure it returns 404? The getting started should display the Welcome index.html page when the root URL is accessed

Reply:
Hum, you may be right, I just remember that it's blue and didn't show my hello world message.
I can remove the sentence if you want.

Reply:
@gastaldi so yes, the getting started page is displayed. I'll remove this sentence.

Reply:
Sentence removed in all places.